README.md
# infostanderaarhus

## Sass -> css

in root run:
```
npm install
```

To get zen-grids v1.5, run:
```
git clone https://github.com/JohnAlbin/compass-zen-grids.git
```

```
gulp sass
```


CHANGELOG.md
# infostander aarhus changelog

## 1.0.0

* Changed texts and added/removed images.
* Added instructions for compiling with node.
* Removed google analytics.


info.php
<?php
  @include $_SERVER['DOCUMENT_ROOT']. '/inc/head.inc';
?>
<body id="top">
<header class="header" role="banner">
  <a href="#top" class="top-link"><i class="icon-arrow-up"></i> Toppen</a>
  <div class="header--inner">
    <a href="#top" class="logo--link"><img src="../images/logo.png" alt="Infostander Aarhus logo" class="logo"></a>
    <nav class="navigation">
      <a href="#top" title="Til toppen" class="navigation--top">Til toppen</a>
      <a href="#info" title="Info">Info</a>
      <a href="#priser" title="Priser">Priser</a>
      <a href="#kontakt" title="Kontakt">Kontakt</a>
    </nav>
  </div>
</header>
<div class="prices">
  <div class="prices--inner">
    <div class="prices--content">
      <h3  id="priser" class="prices--heading">Opsætning og tekniske informationer</h3>

    </div>
  </div>
</div>

<footer class="footer" role="contentinfo">
  <section class="footer--content-primary">
    <img src="/images/address-map.png" class="footer--map" alt="Kort med ITK Designs fysiske placering">
    <h2>ITK Design</h2>
    Møllegade 1<br>
    8000 Aarhus C<br>
    <strong>E-mail:</strong> <a href="mailto:infostander@aarhus.dk">infostander@aarhus.dk</a><br>
    <strong>Telefon:</strong> +45 8940 5000
  </section>
  <section class="footer--content-secondary">
    <h2 class="footer--heading">Kontakt</h2>
    <div class="footer--lead">
      <p>
        Har du spørgsmål så kontakt os.
      </p>
      <p class="footer--focus">Ring <strong>8940 5000</strong></p>
    </div>
  </section>
</footer>
</body>
</html>

config.rb
# Require any additional compass plugins here.

# Set this to the root of your project when deployed:
http_path = "/"
css_dir = "css"
sass_dir = "sass"
images_dir = "images"
javascripts_dir = "scripts"
additional_import_paths = ["/Library/Ruby/Gems/1.8/gems/zen-grids-1.4/stylesheets"]
  
output_style = :compressed
environment = :production

# To enable relative paths to assets via compass helper functions. Uncomment:
relative_assets = true

# To disable debugging comments that display the original location of your selectors. Uncomment:
line_comments = false
color_output = false


# If you prefer the indented syntax, you might want to regenerate this
# project again passing --syntax sass, or you can uncomment this:
# preferred_syntax = :sass
# and then run:
# sass-convert -R --from scss --to sass sass scss && rm -rf sass && mv scss sass
preferred_syntax = :scss


contact_form.php
<?php
$name = strip_tags($_POST["name"]);
$email = strip_tags($_POST["email"]);
$spam_check = $_POST["message"];

$recipient = "infostander@aarhus.dk";

if ( (!empty($name) && !empty($email)) && empty($spam_check) )  {
  $subject = "Jeg vil gerne kontaktes vedr. Infostander Aarhus";

  $message = "<h1>Jeg vil gerne kontaktes</h1>";
  $message.= "<p>Der er kommet en henvendelse på infostanderaarhus.dk, du bedes kontakte:</p>";
  $message.= "<ul>";
  $message.= "<li><strong>Navn:</strong> ". $name ."</li>";
  $message.= "<li><strong>Email:</strong> ". $email ."</li>";
  $message.= "</ul>";

  $header  = "MIME-Version: 1.0" . "\r\n";
  $header .= "Content-type: text/html; charset=utf-8" . "\r\n";
  $header .= "from: Infostander kontaktformular <www@aakb.dk>" . "\r\n";
  $header .= "reply-to:".$email;

  if (mail($recipient, $subject, $message, $header)) {
    print json_encode(array("status" => TRUE, "data" => "Tak for din henvendelse, du h&oslash;rer fra os snarest."));

  } else {
    $response .= "Der skete en fejl og det lykkedes desværre ikke at sende din henvendelse.";
    $response .= "<br /><br />";
    $response .= "<a href=\"#kontakt\">Klik her for at prøve igen</a>. husk at alle felter skal udfyldes.";
    $response .= "<br /><br />";
    $response .= "Hvis du stadig oplever fejl, så kontakt os på p&aring; " . $recipient . ".";

    print json_encode(array("status" => FALSE,"data" => $response));
  }

} else {
  print json_encode(array("status" => FALSE, "data" => "Du skal udfylde alle felter i formularen, prøv venligst igen."));
}

index.php
<?php
@include $_SERVER['DOCUMENT_ROOT'].'/inc/head.inc';
?>
<body id="top">
<header class="header" role="banner">
  <div class="header--inner">
    <a href="#top" class="logo--link"><img src="../images/logo.png"
                                           alt="Infostander Aarhus logo"
                                           class="logo"></a>
    <nav class="navigation">
      <a href="#galleri" title="Galleri & inspiration"
         class="navigation--first">Galleri & inspiration</a>
      <a href="#priser" title="Priser">Priser</a>
      <a href="#info" title="Info">Information</a>
      <a href="#havneskaerm" title="Info">Havneskærm</a>
      <a href="#kontakt" title="Kontakt">Kontakt</a>
      <span class="aak-logo"><img src="../images/aak-logo.png" alt="Aarhus Kommune"></span>
    </nav>
  </div>
</header>
<div class="primary-content">
  <div class="primary-content--inner">
    <img src="/images/infostander.png" class="primary-content--image"
         alt="Billede af infostander">
    <section class="contact-box--content">
      <h1 class="contact-box--heading">Infostander Aarhus</h1>
      <p class="contact-box--lead">Annoncér dit arrangement ved byens indfaldsveje.</p>
      <p class="contact-box--text">Kontakt os i dag og find ud af, hvilke muligheder du har.</p>
      <a href="#kontakt" class="contact-box--link"><i
            class="icon-arrow-right"></i>Kontakt</a>
    </section>
  </div>
</div>
<div class="secondary-content">
  <section class="focus-box--first">
    <a href="#priser" class="focus-box--icon"><i class="icon-tag"></i></a>
    <h3 class="focus-box--heading"><a href="#priser">Priser</a></h3>
    <div class="focus-box--content">
      <p>Der kan købes annoncering i 7 eller 14 dage, inkl. opsætning af grafik.</p>
    </div>
    <a href="#priser" class="focus-box--link"><i
          class="icon-arrow-right"></i>Se priser</a>
  </section>
  <section class="focus-box--second">
    <a href="#info" class="focus-box--icon"><i class="icon-info"></i></a>
    <h3 class="focus-box--heading"><a href="#info">Information</a></h3>
    <div class="focus-box--content">
      <p>Få yderligere information om infostanderne.</p>
    </div>
    <a href="#info" class="focus-box--link"><i class="icon-arrow-right"></i>Information</a>
  </section>
  <section class="focus-box--third">
    <a href="#galleri" class="focus-box--icon"><i class="icon-picture"></i></a>
    <h3 class="focus-box--heading"><a href="#galleri">Inspiration & galleri</a>
    </h3>
    <div class="focus-box--content">
      <p>Se eksempler på tidligere annoncering på infostanderne.</p>
    </div>
    <a href="#galleri" class="focus-box--link"><i class="icon-arrow-right"></i>Inspiration</a>
  </section>
</div>
<div class="gallery">
  <h2 id="galleri" class="gallery--heading">Galleri &amp; inspiration</h2>
  <ul class="gallery--list">
    <li class="gallery--item-first"><a href="gallery/gallery-1.jpg"><img
            src="gallery/gallery-1.jpg"></a></li>
    <li class="gallery--item-second"><a href="gallery/gallery-2.jpg"><img
            src="gallery/gallery-2.jpg"></a></li>
    <li class="gallery--item-third"><a href="gallery/gallery-3.jpg"><img
            src="gallery/gallery-3.jpg"></a></li>
  </ul>
  <div class="gallery-slider">
    <div class="slide"><a href="gallery/gallery-small-1.jpg"><img
            src="gallery/gallery-small-1.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-2.jpg"><img
            src="gallery/gallery-small-2.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-3.jpg"><img
            src="gallery/gallery-small-3.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-5.jpg"><img
            src="gallery/gallery-small-5.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-6.jpg"><img
            src="gallery/gallery-small-6.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-10.jpg"><img
            src="gallery/gallery-small-10.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-11.jpg"><img
            src="gallery/gallery-small-11.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-12.jpg"><img
            src="gallery/gallery-small-12.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-13.jpg"><img
            src="gallery/gallery-small-13.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-14.jpg"><img
            src="gallery/gallery-small-14.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-15.jpg"><img
            src="gallery/gallery-small-15.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-17.jpg"><img
            src="gallery/gallery-small-17.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-18.jpg"><img
            src="gallery/gallery-small-18.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-19.jpg"><img
            src="gallery/gallery-small-19.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-20.jpg"><img
            src="gallery/gallery-small-20.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-21.jpg"><img
            src="gallery/gallery-small-21.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-22.jpg"><img
            src="gallery/gallery-small-22.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-23.jpg"><img
            src="gallery/gallery-small-23.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-24.jpg"><img
            src="gallery/gallery-small-24.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-25.png"><img
            src="gallery/gallery-small-25.png"></a></div>
    <div class="slide"><a href="gallery/gallery-small-26.jpg"><img
            src="gallery/gallery-small-26.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-27.jpg"><img
            src="gallery/gallery-small-27.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-28.jpg"><img
            src="gallery/gallery-small-28.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-29.jpg"><img
            src="gallery/gallery-small-29.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-30.jpg"><img
            src="gallery/gallery-small-30.jpg"></a></div>
    <div class="slide"><a href="gallery/gallery-small-31.jpg"><img
            src="gallery/gallery-small-31.jpg"></a></div>
  </div>
</div>
<div class="prices">
  <div class="prices--inner">
    <i class="prices--content-image icon-tag"></i>
    <div class="prices--content">
      <h2 id="priser" class="prices--heading">Priser</h2>
      <p>
        1 uges visning = 8.000 kr.<br>
        2 ugers visning = 15.000 kr.
      </p>
      <p>Kontakt os for at høre prisen, hvis I har behov for at annoncere i en længere periode.</p>
      <p>Alle priser er eks. moms og inkl. administrationsomkostninger og opsætning af grafik.</p>
      </p>
      <a href="#kontakt" class="prices--contact-link"><i
            class="icon-arrow-right"></i>Kontakt</a>
    </div>
  </div>
</div>
<div class="info">
  <div class="info--content-primary">
    <h3 id="info" class="info--heading">Information</h3>
    <p>Infostanderne i Aarhus kan benyttes til markedsføring af kulturelle og sportslige events, hvor alle byens borgere kan deltage. De digitale infostandere er placeret ved otte af de mest trafikerede indfaldsveje til Aarhus (Grenåvej, Randersvej, Viborgvej, Edwin Rahrs Vej, Silkeborgvej, Skanderborgvej, Christian X’s Vej og Oddervej).</p>
    <p>Infostanderne er synlige for både fodgængere og cyklister, og de er tændt døgnet rundt. Skærmene er 2 meter brede og 3 meter høje, og de lysende LED-skærme sender klare og tydelige budskaber til de mere end 200.000 trafikanter, der kører forbi hver dag.</p>
    <p>De digitale infostandere viser som udgangspunkt rullende 5 spots ad gangen, og de 8 infostandere viser den samme spotrække.</p>
    <p>Fra foråret 2018 er det muligt at få vist en annonce på Havneskærmen, der står i den nordlige del af Havnepladsen mellem Dokk1 og Navitas. Muligheden gælder for de annoncører, der kan benytte de digitale infostandere ved indfaldsvejene. Man vil kunne købe en uges visning på Havneskærmen, og dermed få vist sin annonce for de mange forbipasserende i den centrale del af byen.</p>
    <p>Visningen på Havneskærmen vil dog afhænge af de store events, der i løbet af året finder sted her, og man kan derfor kun booke visning på Havneskærm, såfremt den ikke benyttes af lejere af Havnepladsen.</p>
    <p>Skærmen er 21,6 kvadratmeter, og den er tændt fra kl. 7.00 til kl. 23.00. Billedeslides ligger på skærmen i et rul med andre billedefiler og videomateriale, dels fra andre annoncører dels fra City Lab Aarhus, læs mere: <a href="http://www.smartaarhus.eu/">http://www.smartaarhus.eu/</a></p>
    <p>Kontakt os for at høre nærmere omkring booking og pris.</p>
    <h5>Fordelen ved digitale spots</h5>
    <ul>
      <li>Eksponering for mere end 200.000 trafikanter hver dag</li>
      <li>Visning på 8 strategisk placerede infostandere</li>
      <li>Visning døgnet rundt</li>
      <li>Visning på lysende LED-skærme (2 x 3 meter)</li>
    </ul>
    <h5>Hvem kan annoncere via Infostander Aarhus?</h5>
    <p>Servicen kan benyttes af alle, der arrangerer kulturelle og sportslige events, hvor alle byens borgere kan deltage, uddannelsesinstitutioner i Aarhus samt kommunale kampagner.</p>
    <h5>Retningslinjer for annoncering</h5>
    <ul>
      <li>Events skal understøtte byens brand</li>
      <li>Der kan kun annonceres for events, der afholdes i Aarhus kommune og som Aarhus Kommune har hjemmel til at støtte</li>
      <li>Uddannelsesinstitutioner i Aarhus Kommune kan annoncere på standerne</li>
      <li>De af politiet opsatte retningslinjer skal overholdes</li>
      <li>Spots må ikke indeholde stødende billeder eller tekst</li>
      <li>
        Der må ikke være reklamer eller kommercielle budskaber på spots, men:
        <ul>
          <li>Der må reklameres for et arrangement i byen, hvor der skal betales entré</li>
          <li>Der må henvises til billetsalg vedrørende et arrangement via en hjemmeside eller henvisning til et billetsalgssted</li>
          <li>Der må bruges virksomhedsnavne, hvis det indgår i eventtitel, som f.eks. GF World Cup</li>
          <li>Logoer fra sponsorer må figurere på spots med sekundær placering</li>
        </ul>
      </li>
    </ul>
    <h5>Praktiske forhold</h5>
    <p>Såfremt vi skal sætte grafikken op, skal følgende materiale leveres pr. mail senest 4 uger før spottet skal vises:</p>
    <ul>
      <li>Baggrundsbillede skal du levere i den bedst mulige opløsning (jpg-fil). Beskriv hvilken del af billedet der skal bruges.</li>
      <li>Teksten bør være korte og præcise budskaber. Du skal også her specificere, hvor teksten skal stå, om den skal stå med store eller små bogstaver, og hvordan den skal sættes op; i linier, i en tekstboble, top, bund osv.</li>
      <li>Det væsentlige er: hvad, hvor og hvornår!</li>
      <li>Et eventuelt logo skal du sende som pdf-, eps- eller ai-fil</li>
    </ul>
    <p>Det er også muligt selv at levere det billede, der skal vises.</p>
  </div>
  <div class="info--content-secondary">
    <figure>
      <img src="/images/infostandere-placering.png"
           alt="Kort over placering af infostandere">
      <figcaption>Kort over placering af infostandere ved indfaldsvejene ved Aarhus. Hver dag kører der i gennemsnit 200.000 trafikanter forbi infostanderne.</figcaption>
    </figure>
    <h5>Gode råd, hvis du selv producerer annoncen</h5>
    <ul>
      <li>Skriv korte og enkle budskaber – Det er begrænset hvor meget tekst, de forbipasserende kan nå at læse</li>
      <li>Lys tekst fungerer godt på en mørk baggrund</li>
      <li>Brug stor skriftstørrelse – det varierer fra font til font</li>
      <li>Ønsker du en lys baggrund, så undgå at lave den helt hvid. Vores erfaring viser, at en 100% hvid baggrund kommer til at lyse så kraftigt, at teksten bliver meget utydelig, uanset hvor stor skriften er</li>
      <li>Når du har produceret din annonce er det en god idé at se på det i halv størrelse på computerskærmen. Hvis spottet fungerer i denne størrelse, fungerer det med stor sandsynlighed også ude på de store infostanderne</li>
    </ul>
    <h5>Krav til billedfiler, Digitale Infostandere:</h5>
    <ul>
      <li>Filformat: JPEG</li>
      <li>Opløsning: 256 x 384 pixels – 72 ppi</li>
      <li>Farver: RGB</li>
    </ul>
    <h5>Krav til billedefiler, Havneskærm:</h5>
    <ul>
      <li>Filformat: JPEG</li>
      <li>Opløsning: 900 x 540 pixels (16/9-format)</li>
      <li>Farver: RGB</li>
    </ul>
    <p>Har du spørgsmål eller ønsker du at booke annoncering, så <a
          href="#kontakt" title="Kontakt os">kontakt os</a>!</p>
    <p>
      <i>Infostander Aarhus er et initiativ iværksat af Aarhus Kommune med henblik på at sikre eksponeringsmuligheder for byens eventoperatører, og derved styrke Aarhus som en førende Eventby.</i>
    </p>
    <p>
      <i>Priserne er fastsat med henblik på at dække omkostninger til etablering og drift af infostanderne, samt løbende administrationsomkostninger, herunder markedsføring, salg, grafisk arbejde og systemadministration. Et eventuelt overskud skal anvendes til at understøtte kulturelle formål i byen.</i>
    </p>
    <p>
      <i>Aarhus Kommune forbeholder sig rettigheden til at bruge muligheden for at vise en 6. annonce, såfremt der er brug for det. Det 6. spor vil udelukkende blive brugt til kommunale kampagner med grundlæggende oplysninger til borgerne. Der vil maks. blive vist 6 annoncer på standerne og min. 3 annoncer.</i>
    </p>
  </div>
</div>
<div class="info">
  <div class="info--content-primary">
    <h3 id="havneskaerm" class="info--heading">Havneskærm for lejere af Havnepladsen</h3>
    <p>Som lejer af Havnepladsen i det centrale Aarhus kan man leje Havneskærmen, der er placeret i den nordlige ende af pladsen.</p>
    <p>Skærmen er 21,6 kvadratmeter og den er tændt fra kl. 7.00 til kl. 23.00. Afvikles der aktiviteter udenfor dette tidsrum kan der søges lejlighedstilladelse. Skærmen er en LED skærm og har automatisk regulering af lysstyrke baseret på luxmåling i omgivelserne.</p>
    <p>Man kan leje skærmen som en ekstern skærm, eller man kan sende de færdigproducerede billeder og video til os, og så lægger vi det på skærmen i en aftalt periode.</p>

    <br/><br/>

    <p>Der er følgende muligheder:</p>

    <h5>Løsning 1 – visning via Aarhus Kommunes storskærmskanal</h5>
    <p>Man kan få vist sine billeder og video via Aarhus Kommunes storskærmskanal. Materialet sendes til os inkl. beskrivelse af rækkefølge.</p>
    <p>Materialet skal leveres i:</p>
    <p>Video kan leveres i 720p eller 1080p, en mindre opløsning kan også fungere.</p>
    <p>Billeder ses bedst i 16:9 ration/ 1920x1080px</p>

    <br/><br/>

    <h5>Løsning 2 – visning via tilkobling af eget udstyr, uden lyd</h5>
    <p>Tilkobling af egen computer er muligt via vores AV-RACK.</p>
    <p>Video input: et HDMI-kabel er tilstede. Hvis den medbragte computer ikke understøtter HDMI, skal man selv medbringe en konverter.</p>
    <p>Opløsning: op til 4K. Output signal fra den medbragte computer bliver scaleret ned til 900x540 pixels via vores AV-RACK.</p>

    <br/><br/>

    <h5>Løsning 3 – tilkobling af eget udstyr, med lyd.</h5>
    <p>Tilkobling af egen computer er muligt via vores AV-RACK.</p>
    <p>Video input: HDMI-kabel er tilstede. Hvis den medbragte computer ikke understøtter HDMI, skal man selv medbringe en konverter.</p>
    <p>Opløsning: op til 4K. Output signal fra den medbragte computer bliver scaleret ned til 900x540pixels via vores AV-RACK.</p>
    <p>Audio input: 3,5 line-in jackstik.</p>
    <p>Mikrofoner: 1 stk. håndholdt mikrofon. 1 stk. trådløst headset.</p>
    <p>Der er ekstra omkostninger forbundet med opsætning af højtalere og lyd.</p>

    <br/><br/>
    <p>Er der specielle ønsker, som ikke er beskrevet her, så kontakt os.</p>
  </div>
  <div class="info--content-secondary">
    <h5>Reglement for anvendelse af Havneskærmen på Havnepladsen</h5>

    <p>For indholdet på Havneskærmen gælder følgende overordnede regler:</p>
    <ul>
      <li>De af politiet opsatte retningslinjer skal overholdes.</li>
      <li>Der må ikke vises stødende billeder, tekst eller spots/film på skærmen.</li>
      <li>
        Skærmen må ikke gøres til brug for reklamering af kommerciel art, og således kun bruges til formidling af aktiviteter, der understøtter anvendelse af pladsen (jf. bl.a. lokalplan 876 § 9 stk. 1).
        <br/>Dog må der:
        <ul>
          <li>Reklameres for et arrangement i byen, også hvor der skal betales entré.</li>
          <li>Henvises til billetsalg vedrørende et arrangement via en hjemmeside eller henvisning til et billetsalgssted.</li>
          <li>Bruges virksomhedsnavne, hvis det indgår i eventtitel, som f.eks. Volvo Ocean Race.</li>
          <li>Figurere logoer fra sponsorer med sekundær placering.</li>
        </ul>
      </li>
      <li>Brug af skærmen knytter sig til eventen og ikke til lejeperioden af Havnepladsen.</li>
      <li>Aktører som lejer pladsen og som ønsker at benytte skærmen, betaler leje svarende til markedsprisen. For arrangementer som Aarhus Kommune har hjemmel til at støtte, fastsættes lejen til et beløb svarende til den faktiske økonomiske omkostning (afskrivning, forbrug og medgået tid).</li>
      <li>Skærmen vil som udgangspunkt være slukket i perioden kl. 23.00 til kl. 7.00, hvorfor der ved aktiviteter uden for dette tidsrum skal søges om lejlighedstilladelse.</li>
      <li>Skærmen må bruges til formidling af aktiviteter som debatmøder, oplæsning af litteratur, digital ’Speakers Corner’, politiske debatter, foredrag, digitale afstemninger/polls, digitalt mødested, ’Silent Concerts’, digitale cykelture, høringer om f.eks. byudvikling, sportsaktiviteter samt understøttelse af andre aktiviteter på pladsen og i området som f.eks. sportsevents, food events, udstillinger og markeder.</li>
    </ul>

    <h5>Krav til billedefiler:</h5>
    <ul>
      <li>Filformat: JPEG</li>
      <li>Opløsning: 900 x 540 pixels (16/9-format)</li>
      <li>Farver: RGB</li>
    </ul>

    <h5>Krav til videomateriale:</h5>
    <ul>
      <li>Opløsning på skærmen er 900x540 pixels.</li>
      <li>Skærmen understøtter video opløsning i 720p (1280x720) og 1080p (1920x1080).</li>
    </ul>

    <h5>Priser</h5>
    <p>Prisen opdeles i to kategorier, jf. reglement:</p>
    <p>For lejere af Havnepladsen, som Aarhus Kommune ikke har hjemmel til at støtte er prisen for leje af Havneskærm pr. døgn: 13.000kr. (eks. moms)</p>
    <p>For lejere af Havnepladsen, som Aarhus Kommune via Kommunalfuldmagten har hjemmel til at støtte, er prisen den faktiske økonomiske omkostning. Kontakt os for at høre nærmere.</p>

    <br/><br/>
    <p>
      <i>Reklameringen på digitale infostandere og på Havneskærmen sker i overensstemmelse med lovgivningen, herunder markedsføringsloven, hvilket fremgår af § 2 i lov nr. 490 af 7. juni 2006 om kommuners og regioners anvendelse af fast ejendom og løsøre til brug for reklamering for andre.</i>
    </p>
  </div>
</div>
<footer class="footer" role="contentinfo">
  <section class="footer--content-primary">
    <img src="/images/address-map.png" class="footer--map"
         alt="Kort med ITK Designs fysiske placering">
    <img src="../images/aak-logo-big.png" alt="Aarhus Kommune"
         class="aak-logo-big">
    <h2>ITK</h2>
    Aarhus Kommune<br>
    Hack Kampmanns Plads 2<br>
    8000 Aarhus C<br>
    <strong>E-mail:</strong> <a
        href="mailto:infostander@aarhus.dk">infostander@aarhus.dk</a><br>
    <strong>Telefon:</strong> +45 4185 6563
  </section>
  <section class="footer--content-secondary" id="kontakt">
    <h2 class="footer--heading">Kontakt</h2>
    <div class="footer--lead">
      <p>
        Kontakt os for at booke annoncering på infostandere eller leje Havneskærmen.
      </p>
      <p class="footer--focus">Ring <a
            href="tel:41856563"><strong>4185 6563</strong></a> eller udfyld formularen
      </p>
      <p>
        Så vender vi tilbage til dig hurtigst muligt.
      </p>
    </div>
    <div class="message"></div>
    <form action="contact_form.php" method="post" id="contact-form">
      <label for="name">Navn:</label>
      <input type="text" id="name" name="name" class="contact-form--name"
             placeholder="Indtast dit navn" required>
      <label for="email">E-mail:</label>
      <input type="email" id="email" name="email" class="contact-form--email"
             placeholder="Indtast din e-mail-adresse" required>
      <input type="hidden" id="message" name="message" value="">
      <button class="contact-form--button" type="submit"
              id="contactSubmitButton">
        <i class="icon-arrow-right"></i>
        Send
      </button>
    </form>
  </section>
</footer>

<script type="text/javascript">
    function updateMessage (target, status) {
        target
            .removeAttr('class')
            .addClass(status);
    }

    $(document).ready(function () {
        var target = $('.message');

        $('#contactSubmitButton').click(function (e) {
            $.ajax({
                type: 'POST',
                url: 'contact_form.php',
                data: $('#contact-form').serialize(), // serializes the form's elements.
                dataType: 'json'
            }).done(function (data) {
                if (data.status) {
                    updateMessage(target, 'message--success');
                }
                else {
                    updateMessage(target, 'message--error');
                }

                target.html(data.data);
            });

            e.preventDefault();
            return false; // avoid to execute the actual submit of the form.
        });
    });
</script>
</body>
</html>
